test/IISIntegration.FunctionalTests/Inprocess/StartupExceptionTests.cs

```diff
@@ -29,6 +29,8 @@ public class StartupExceptionTests : IISFunctionalTestBase
 
             Assert.Equal(HttpStatusCode.InternalServerError, response.StatusCode);
 
+            Dispose();
```

Comment:
Add `StopServer` method to a base class that would dispose deployer and set it to null. Not a fan of disposing `this` and it would prevent double dispose.